collect_app/src/main/java/org/odk/collect/android/utilities/HtmlUtils.java

```diff
@@ -77,32 +77,40 @@ static String markdownToHtml(String text) {
                 text, createSpan);
 
         //intermediary replacements keys for special characters, N/B: These symbols are not meant to be interpreted as markdown
-        text = text.replaceAll("(?s)\\\\#", "&#35;");
-        text = text.replaceAll("(?s)\\\\\\\\", "&#92;");
-        text = text.replaceAll("(?s)\\\\_", "&#95;");
-        text = text.replaceAll("(?s)\\\\\\*", "&#42;");
+        text = text.replaceAll("\\\\#", "&#35;");
+        text = text.replaceAll("\\\\\\\\", "&#92;");
+        text = text.replaceAll("\\\\_", "&#95;");
+        text = text.replaceAll("\\\\\\*", "&#42;");
 
         // strong
         text = text.replaceAll("(?s)__(.*?)__", "<strong>$1</strong>");
         text = text.replaceAll("(?s)\\*\\*(.*?)\\*\\*", "<strong>$1</strong>");
 
-        // emphasis
-        text = text.replaceAll("(?s)_([^\\s][^_\n]*)_", "<em>$1</em>");
-        text = text.replaceAll("(?s)\\*([^\\s][^\\*\n]*)\\*", "<em>$1</em>");
+        // emphasis using underscore
+        text = text.replaceAll("\\s_([^\\s][^_\n]*)_\\s", " <em>$1</em> ");
+        text = text.replaceAll("^_([^\\s][^_\n]*)_$", "<em>$1</em>");
+        text = text.replaceAll("^_([^\\s][^_\n]*)_\\s", "<em>$1</em> ");
+        text = text.replaceAll("\\s_([^\\s][^_\n]*)_$", " <em>$1</em>");
+
+        // emphasis using asterisk
+        text = text.replaceAll("\\s\\*([^\\s][^\\*\n]*)\\*\\s", " <em>$1</em> ");
```

Comment:
I think we actually want to keep `*` the same. There's no practical use of `*` surrounded by text that I can think of. This is why Github makes the space special behavior only for `_`:

`un*frigging*believable`: un*frigging*believable
`un_frigging_believable`: un_frigging_believable

Reply:
Ok I've reverted that change.